datafusion/Cargo.toml

```diff
@@ -68,7 +68,7 @@ md-5 = { version = "^0.10.0", optional = true }
 sha2 = { version = "^0.10.1", optional = true }
 blake2 = { version = "^0.10.2", optional = true }
 blake3 = { version = "1.0", optional = true }
-ordered-float = "2.0"
+ordered-float = "2.10"
```

Comment:
Strange that the Dependbot didn't update this :(

Reply:
It's because not a "real" upgrade and we don't have a `Cargo.lock` file.

A `cargo update` will also have upgraded the version to the latest version, e.g. https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements